mint-auditor/migrations/2022-05-24-214504_create_initial/up.sql

```diff
@@ -35,7 +35,7 @@ CREATE TABLE mint_config_txs (
     -- The token id this mint config tx is for.
     token_id BIGINT NOT NULL,
     -- The nonce, as hex-encoded bytes.
-    nonce_hex VARCHAR(128) NOT NULL UNIQUE,
+    nonce_hex VARCHAR(128) NOT NULL,
```

Comment:
These mint-auditor changes need to be coordinated with @david1alvarez 

This is not the real mint-auditor anymore, this is just here for testing.
See README of mint-auditor crate

Reply:
I'm not sure if it's necessary to change the mint auditor at all in this PR, will it actually break if we don't change it?

Reply:
test_find_unaudited_mint_tx_by_nonce seems to fail when I remove this change. I'll coordinate with @david1alvarez.

Reply:
It will break if at some point the nonce is not unique, the unique constraint now needs to be on `(token_id, nonce_hex)`. If we are only planning on using the mint-auditor here for internal testing then that's not a problem, but the reserve auditor should alter the unique constraint.